docs/4.0/components/alerts.md

```diff
@@ -10,38 +10,38 @@ toc: true
 
 Alerts are available for any length of text, as well as an optional dismiss button. For proper styling, use one of the eight **required** contextual classes (e.g., `.alert-success`). For inline dismissal, use the [alerts jQuery plugin](#dismissing).
 
-{% example html %}
+{% capture example %}
 {% for color in site.data.theme-colors %}
 <div class="alert alert-{{ color.name }}" role="alert">
   A simple {{ color.name }} alert—check it out!
 </div>{% endfor %}
-{% endexample %}
+{% endcapture %}{% include example.html content=example %}
```

Comment:
Is there any reason you have added the `include`s to the same line?

Reply:
No reason, just easier the first search-and-replace batch - I've change it if you want